shop-the-look/.env.development.example

```diff
@@ -0,0 +1,4 @@
+GOOGLE_CREDENTIALS_BASE64=
```

Comment:
SUPER NIT: could consider adding comments above each of these explaining where to get the values, e.g, https://app.pinecone.io for PINECONE_API_KEY...